doc/user_guide/getting_started.rst

```diff
@@ -27,25 +21,35 @@ started as follows:
 
     $ jupyter notebook
 
-Some may find it more convenient to start Jupyter/IPython from the `file manager
-context menu <https://github.com/hyperspy/start_jupyter_cm>`_ or by
-`double-clicking a notebook file <https://github.com/takluyver/nbopen>`_.
+Linux users may find it more convenient to start Jupyter/IPython from the `file manager
+context menu <https://github.com/hyperspy/start_jupyter_cm>`_ . In either OS you can also start by
+`double-clicking a notebook file <https://github.com/takluyver/nbopen>`_ if one already exists.
 
+Starting HyperSpy in the notebook (or terminal)
+-----------------------------------------------
 Typically you will need to `set up IPython for interactive plotting with
 matplotlib
-<http://ipython.readthedocs.org/en/stable/interactive/plotting.html>`_ using the
-``%matplotlib`` magic *before executing any plotting command*. So, typically,
-after starting IPython, you can import
-HyperSpy and set up interactive matplotlib plotting by executing the following
+<http://ipython.readthedocs.org/en/stable/interactive/plotting.html>`_ using 
+``%matplotlib`` (which is known as a 'Jupyter magic') *before executing any plotting command*. So, typically,
+after starting IPython, you can import HyperSpy and set up interactive matplotlib plotting by executing the following
 two lines in the IPython terminal:
 
 .. code-block:: python
 
    In [1]: %matplotlib qt
    In [2]: import hyperspy.api as hs
 
+Note that to execute lines of code in the notebook you must press ``shift+return``. (For details about notebooks and their functionality try the help menu in the notebook).
```

Comment:
``Shift+Return``, i.e. capitalized, could be argued as the convention, on Windows at least? Minor point I know...